sdks/python/apache_beam/examples/snippets/snippets.py

```diff
@@ -1394,28 +1397,24 @@ def __init__(self, string_vp):
     # The DoFn is called when creating the pipeline branch.
     # This example logs the ValueProvider value, but
     # you could store it by pushing it to an external database.
-    def process(self, an_int):
+    def process(self, an_int, **kwargs):
       logging.info('The string_value is %s' % self.string_vp.get())
-      # Another option (where you don't need to pass the value at all) is:
-      logging.info('The string value is %s' %
-                   RuntimeValueProvider.get_value('string_value', str, ''))
 
   pipeline_options = PipelineOptions()
-  # Create pipeline.
-  p = beam.Pipeline(options=pipeline_options)
 
   my_options = pipeline_options.view_as(MyOptions)
-  # Add a branch for logging the ValueProvider value.
-  _ = (p
-       | beam.Create([None])
-       | 'LogValueProvs' >> beam.ParDo(
-           LogValueProvidersFn(my_options.string_value)))
-
-  # The main pipeline.
-  result_pc = (p
-               | "main_pc" >> beam.Create([1, 2, 3])
-               | beam.combiners.Sum.Globally())
 
-  p.run().wait_until_finish()
-
-  # [END AccessingValueProviderInfoAfterRunSnip1]
+  # Create pipeline.
+  with beam.Pipeline(options=my_options) as p:
+    # Add a branch for logging the ValueProvider value.
+    _ = (p
+         | beam.Create([None])
+         | 'LogValueProvider' >> beam.ParDo(
+              LogValueProvidersFn(my_options.string_value)))
+
+    # The main pipeline.
```

Comment:
Does it make sense to completely move the snippet to the `snippets_test.py` file, and remove it from snippets.py?

Reply:
Sure, I've moved it to snippets_test.py